README.md

```diff
@@ -9,7 +9,7 @@ https://www.youtube.com/watch?v=UjsQFNN0nSA
 
 ### Plain instructions
 
-1. Make sure you've installed [Python 3.6 64 bit](https://www.python.org/ftp/python/3.6.5/python-3.6.5-amd64.exe). During installation:
+1. Make sure you've installed [Python 3.7](https://www.python.org/ftp/python/3.7.1/python-3.7.1-amd64.exe) or [Python 3.6 64 bit](https://www.python.org/ftp/python/3.6.5/python-3.6.5-amd64.exe) (deprecated). During installation:
```

Comment:
Let me know your thoughts.

Reply:
I agree with linking to the downloads page rather than linking to the direct download. We don't want people installing older versions of a release when there's no reason to.